### PR TITLE
Updated index.html (Added new feature: "Hyperlinked")

### PR DESCRIPTION
**Pull request by yashk0thari**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x} Included a description of my change below

# Things done in this Pull Request
solved issue #523 

- Replace this with your change description

### CODE REVIEW

**Comment by yashk0thari** [comment]
https://beta-king.github.io/Yash-Kotnari.gci17.fossasia.org/ --> preview links

File: index.html [hunk 1]
```diff
@@ -342,19 +342,19 @@ <h2>Open Event Organizer App</h2>
...
 			<div class="col-sm-4">
-				<h2 class="number">370+ students</h2>
+				<a href="https://gci17.fossasia.org/students"><h2 class="number">370+ students</h2></a>
 				<p>
 					Over 370 students participated in Fossasia Google code-in 2017, many students made their way in contributing on many open source projects.
 				</p>
 			</div>
 			<div class="col-sm-4">
-				<h2 class="number">300+ Tasks</h2>
+				<a href="https://codein.withgoogle.com/tasks/?sp-organization=6237166344601600"><h2 class="number">300+ Tasks</h2></a>
```
Comment (anshumanv) [inline index.html]: Replace link with this - https://codein.withgoogle.com/organizations/fossasia/

Comment (yashk0thari) [reply index.html]: ohkk

**Comment by yashk0thari** [comment]
Made necessary changes

**Comment by yashk0thari** [comment]
please review it guys...

**Review by ParthS007 — APPROVED** [review]
LGTM 👍

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Please squash your commits next time.

**Pull request merged**